code/__HELPERS/AStar.dm

```diff
@@ -189,6 +189,7 @@ Actual Adjacent procs :
 	if(T && !T.density && !(simulated_only && SSpathfinder.space_type_cache[T.type]) && !LinkBlockedWithAccess(T,caller, ID))
 		return TRUE
 
+
```

Comment:
?

Reply:
Likely that this file was changed then the changes were undone but not cleanly. Run `git checkout upstream/master -- code/__HELPERS/AStar.dm` in git CLI to fix